containers/php/fpm/docker-php-entrypoint

```diff
@@ -20,7 +20,7 @@ fi
 
 # first arg is `-f` or `--some-option`
 if [ "${1#-}" != "$1" ]; then
-	set -- php-fpm "$@"
+  set -- php-fpm "$@"
```

Comment:
I recommend to only indent with tabs in shell scripts, as there are places where only tabs are allowed as indent and spaces can't be used (https://unix.stackexchange.com/a/76483/291714). Sticking to tabs prevents mixed indentation characters.

Reply:
Done